articles/arm-x86/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/arm-x86/
+title: ARM vs x86 - Explained
+description: 
+author: mike-white
+date: 2020-09-10T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/arm-x86/hero.jpg
+    alt: 
+---
+Recently, Apple [announced their decision to switch from Intel CPUs to their own ARM chips](https://www.theverge.com/2020/6/22/21295475/apple-mac-processors-arm-silicon-chips-wwdc-2020). What's the difference? The new Apple Silicon chips are based on ARM CPUs, like the CPUs for iPhone and iPad. The Intel chips use Intel's proprietary x86 architecture.
+<!--more-->
+
+ARM is a RISC architecture, [which we've talked about before](https://www.section.io/engineering-education/what-is-risc/). RISC stands for Reduced Instruction Set Computing. This means that the CPU has a limited number of instructions it can use. As a result, each instruction runs in a single cycle and the instructions are simpler. Meanwhile, x86 is a CISC architecture, which stands for Complex Instruction Set Computing. This means that it has more instructions. Some of these instructions will take more than one cycle to execute. But, there are some instructions that can do the work of many RISC instructions.
+
+Another benefit of ARM is that it's a semi-open architecture. [Very few companies manufacture x86 processors](https://en.wikipedia.org/wiki/List_of_x86_manufacturers), because it's owned by Intel. Intel doesn't want too much competition. ARM doesn't actually manufacture their own CPUs. [They license the design out to other companies](https://developer.arm.com/support/licensing) who want to make their own CPUs. [Apple is one of them](https://en.wikipedia.org/wiki/Apple_Silicon). Apple is also able to customize their chips to work better on their platforms. That's presumably why [iPhones perform very well in benchmarks](https://benchmarks.ul.com/compare/best-smartphones?amount=200&sortBy=PERFORMANCE&reverseOrder=true&osFilter=ANDROID,IOS&test=SLING_SHOT_ES_30_UNLIMITED&deviceFilter=PHONE&displaySize=3.0,15.0).
```

Comment:
Rather than saying Intel doesn't want competition, you can contrast this to semi-open by saying it is closed source- and further define both those terms for the reader.